RELEASE_NOTES.md

```diff
@@ -19,6 +19,8 @@
 - You can no longer use functions like `CheckoutConfiguration.getCardConfiguration()` or `CheckoutConfiguration.getDropInConfiguration()` to get configurations from the  `CheckoutConfiguration` object. When starting Drop-in or Components, pass the full `CheckoutConfiguration` object.
 
 ## Changed
+- Top navigation has been added to Drop-in and dragging gesture has been disabled which caused Drop-in to dismiss.
```

Comment:
The existing release notes are from the latest release, we need to clear the file